Lobi sept17

JosuaHaug opened the pull request.

Commit:
Closes #146

Commit:
Adjust key of king of my heart

Commit:
Closes #145

Commit:
Adjusted schedule for no longer slaves

Commit:
Closes  #143

Add new song "Wir sind Eins" by Outbreakband

Commit:
Closes #142

Add new song "Mehr als je zuvor"

Commit:
Closes #144

Add new song "Bis ich dir gegenüber steh"

nivekznews commented on King_of_my_Heart.tex:
```diff
@@ -0,0 +1,64 @@
+\end{bridge}
+
+\begin{bridge}
+^{Em}\_ You're ^{D}never gonna let | ^{C}never gonna let ^{G}me down | \\
```
Didn't we simplify the bridge's chords?

JosuaHaug replied:
Has been fixed with new bridges and Schedule (reference: https://youtu.be/EpqSbKYxd9Y)

nivekznews commented on No_longer_slaves.tex:
```diff
@@ -0,0 +1,50 @@
+\begin{verse}             
+From my Moth- | ^{A}ers womb You have | ^{C#m}chosen me 
+| ^DLove has ^Ecalled my | ^Aname \\
+I’ve been |^Aborn again, into your | ^{C#m}family 
```
Should be:
I've been born a- | ^Again, into your fami- | ^{C#m}ly;

JosuaHaug replied:
Has been fixed in new commit

nivekznews commented on Wir_sind_Eins_(Outbreakband).tex:
```diff
@@ -0,0 +1,52 @@
+\end{interlude}
+
+\begin{bridge}
+| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Leben gibt \\
```
| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Le- | ben gibt \\
| ^{G}\quarterrest~ Ein Leib | \quarterrest~ ein Geist | ^{A}\quarterrest~ ein Gott der uns | \_ vereint \\
| ^{G}\quarterrest~ Ein Weg | \quarterrest~ ein Ziel | ^{A}\quarterrest~ ein Gott der Le- | ben gibt \\
| ^{G}\quarterrest~ Ein Leib | \quarterrest~ ein Geist | ^{A}\quarterrest~ ein Gott \quarterrest~ | \halfrest~

JosuaHaug replied:
Has been fixed in new commit

nivekznews reviewed (changes requested):
I have have found a few minor issues. Who should fix them?

Commit:
Fix for Review finding #147

King of my heart: Bridge Chords, Schedule
No longer slaves: Measure Bars Verse
Wir sind eins: Measure Bars Bridge

JosuaHaug commented:
Minor issues have been fixed.

Commit:
Merge branch 'master' into lobiSept17

Commit:
Add new stylesheet to feature branch

Commit:
Fix wrong search replace
